### ISSUE TITLE
Scanning for devices cuts the device's name

### ISSUE THREAD

**Issue by StefanGeorgievski** [iDevicesInc/SweetBlue#430]
When scanning for devices  their names get cut after the 17th character.
Eg. My device's name is: 12345678901234567890, but the device I get from the scan is the following:

![image](https://user-images.githubusercontent.com/26904243/38367084-00f2ab84-38e2-11e8-91f9-56d422eb3b3d.png)

In the issues I read that the default MTU (maximum transmission unit) is 23 bytes which is the problem that I probably have.

Is there any way to extend this while scanning for devices so that I can get the device's full name?

Thanks in advance.

**Comment by ryanidev** [comment]
No, the scan record has nothing to do with the MTU. On Bluetooth 4, you only have so much room to put data. Things like advertising flags, manufacturer id/data, service uuids/data. So, I'm guessing your device has some other data in the scan record, leaving 17 bytes for the name.
